.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,129 @@
+on: [push]
+name: "Docker build"
+jobs:
+    build-docker-images:
+        name: Build PHP-FPM and Elasticsearch images
+        runs-on: ubuntu-20.04
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v2
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Login to Docker Hub
+                uses: docker/login-action@v1
+                with:
+                    username: ${{ secrets.DOCKER_USERNAME }}
+                    password: ${{ secrets.DOCKER_PASSWORD }}
+            -   name: Check if PHP-FPM image exists
+                env:
+                    DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+                run: |
+                    DOCKER_PHP_FPM_IMAGE_TAG=gitlab-action-`find project-base/docker/php-fpm -type f -exec stat -c "%a %n" {} \; -exec cat {} \; | md5sum | awk '{ print $1 }'`
+                    echo "DOCKER_PHP_FPM_IMAGE_TAG=${DOCKER_PHP_FPM_IMAGE_TAG}" >> $GITHUB_ENV
+                    echo "DOCKER_USERNAME=${DOCKER_USERNAME}" >> $GITHUB_ENV
+                    DOCKER_PHP_FPM_IMAGE_EXISTS=`. .github/check-image-exists.sh && checkImageExists ${DOCKER_USERNAME} php-fpm ${DOCKER_PHP_FPM_IMAGE_TAG} > /dev/null ; echo $?`
```

Comment:
Using the reponse status code could do it more readable and simple 🤔

Reply:
Sorry I do not agree, I prefer naming variable and then compare it